### PR TITLE
E_CLASSROOM-337 [FE/BE] Fix change category in add/edit question

### PR DESCRIPTION
**Pull request by karlolee50**
### Issue Link
https://framgiaph.backlog.com/view/E_CLASSROOM-337

### Definition of Done
- [x] User can change category of the quiz
- [x] Add/edit question functionality is not broken

### Commands to Run
N/A

### Related PR
fe: https://github.com/framgia/sph-classroom-els-fe/pull/152
#### Main note
- N/A
#### Pages Affected
- http://localhost:82/api/v1/admin_edit_question

#### Scenarios/ Test Cases
- [ ] User can change category of the quiz
- [ ] Add/edit question functionality is not broken

### Screenshots
Refer to my FE PR

### CODE REVIEW

File: app/Http/Controllers/API/v1/Quiz/QuestionController.php [hunk 1]
```diff
@@ -136,9 +136,18 @@ public function editQuestion(Request $request)
...
+        $this->changeCategory($quiz_id, $request->categoryId);
 
         $questions = Question::where('quiz_id', $quiz_id)->get();
 
         return $this->successResponse($questions, 200);
     }
+
+    public function changeCategory($quiz_id, $category_id)
+    {
+        $quiz = Quiz::where('id', $quiz_id)->first();
```
Comment (karlolee50) [inline app/Http/Controllers/API/v1/Quiz/QuestionController.php]:
```suggestion
        $quiz = Quiz::find($quiz_id);
```

**Review by karlolee50 — COMMENTED** [review]
Review time: 5 minutes
Comment/s: 1

**Comment by catampongan** [comment]
## Working
- [x] Able to change the category of the quiz
- [x] Add/edit button is functional
- [x] Redirected to quizzes after saving the changes

## Comment
- [ ] Root location is not working
------------------------------
task-337
task-337
Task-337-1

--------------------------------
- Total Test Time: 0.5 hr

**Comment by catampongan** [comment]
## Approved PR #79 
- [x] Able to change the category of the quiz
- [x] Add/edit button is functional
- [x] Redirected to quizzes after saving the changes
---------------------------------
task-337
Task-337-1
task-337

-----------------------
- Total Test Time: 0.75 hr
- Total Iterations: 1
- Total Comments: 1

**Pull request merged**